clients/src/main/java/org/apache/kafka/common/security/kerberos/KerberosLogin.java

```diff
@@ -213,6 +213,7 @@ public LoginContext login() throws LoginException {
                             break;
                         } catch (Exception e) {
                             if (retry > 0) {
+                                log.warn("[Principal={}]: Error when trying to renew with TicketCache, but will retry ", principal, e);
```

Comment:
@cnZach Looks like the intention was to print out the exception with stack trace? You would want to use:
```
public void warn(String msg, Throwable t);
```

Reply:
Hi @rajinisivaram , do you mean we should use  `log.warn(s"[Principal=$principal]: Error when trying to renew with TicketCache, but will retry ", e); ` ?

Reply:
slf4j logger methods logs exception if the last argument is throwable. so this is fine.